wsclient/src/main/java/org/hpccsystems/ws/client/HPCCFileSprayClient.java

```diff
@@ -1535,6 +1549,31 @@ public boolean uploadLargeFile(File uploadFile, DropZoneWrapper dropZone)
             }
         }
 
+        if (response.length() > 0)
+        {
+            try
+            {
+                DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
```

Comment:
@rpastrana having to parse a raw XML document here seems brittle, but is probably the best we can do atm.